web/admin/src/App.js

```diff
@@ -135,6 +144,12 @@ export default class App extends PureComponent {
     }
   }
 
+  updateUserData = (content) => {
+    const userContent = content.sort(sortContent)
```

Comment:
where is `sortContent` defined?

Reply:
This is a constant at the bottom of the page